src/steps/malwares/index.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  IntegrationStep,
+  IntegrationStepExecutionContext,
+} from '@jupiterone/integration-sdk-core';
+import { createAPIClient } from '../../client';
+import { IntegrationConfig } from '../../config';
+import { Entities, IntegrationSteps } from '../constants';
+import { createMalwareEntity } from './converters';
+
+export async function fetchMalwares({
+  instance,
+  jobState,
+}: IntegrationStepExecutionContext<IntegrationConfig>) {
+  const apiClient = createAPIClient(instance.config);
+  await apiClient.iterateMalwares(async (malware) => {
+    const malwareEntity = createMalwareEntity(malware);
+    await jobState.addEntity(malwareEntity);
+  });
+}
+
```

Comment:
This step throws a DuplicateKey error. You will see it when you run yarn start. Apparently the API is sending back duplicate records.